contracts/lib/ConsiderationBase.sol

```diff
@@ -184,7 +193,7 @@ contract ConsiderationBase is ConsiderationEventsAndErrors {
                 "bytes32 zoneHash,",
                 "uint256 salt,",
                 "bytes32 conduitKey,",
-                "uint256 nonce",
+                "uint256 counter",
```

Comment:
[Documentation](https://docs.opensea.io/v2.0/reference/seaport-overview) still contains old name  (`nonce` instead of `coutner`)